src/transaction/log_page_buffer.c

```diff
@@ -8714,6 +8721,8 @@ logpb_restore (THREAD_ENTRY * thread_p, const char *db_fullname, const char *log
 		case LOG_DBLOG_ARCHIVE_VOLID:
 		case LOG_DBTDE_KEYS_VOLID:
 
+		  is_prev_volheader_restored = false;
```

Comment:
`is_prev_volheader_restored` is always false. 
I think `is_prev_volheader_restored` need to be set true at the part where the previous volume header is restored.

Reply:
See the fileio_restore_volume(). The is_prev_volheader_restored is set in it when matching some condition.

Reply:
Ah, this is in the while loop. I understand.